src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -11,462 +11,374 @@
-                                      {:desc    "Open file in parent directory"
-                                       :binding false
-                                       :fn      page-handler/open-file-in-directory})
+   :editor/open-file-in-directory   {:binding false
+                                     :inactive (not (util/electron?))
+                                     :fn      page-handler/open-file-in-directory}
 
-   :editor/copy-current-file        (when (util/electron?)
-                                      {:desc    "Copy current file"
-                                       :binding false
-                                       :fn      page-handler/copy-current-file})
+   :editor/copy-current-file        {:binding false
+                                     :inactive (not (util/electron?))
+                                     :fn      page-handler/copy-current-file}
 
-   :ui/toggle-wide-mode             {:desc    "Toggle wide mode"
-                                     :binding "t w"
+   :ui/toggle-wide-mode             {:binding "t w"
                                      :fn      ui-handler/toggle-wide-mode!}
 
-   :ui/select-theme-color           {:desc    "Select available theme colors"
-                                     :binding "t i"
+   :ui/select-theme-color           {:binding "t i"
                                      :fn      plugin-handler/show-themes-modal!}
 
-   :ui/goto-plugins                  (when plugin-handler/lsp-enabled?
-                                       {:desc    "Go to plugins dashboard"
-                                        :binding "t p"
-                                        :fn      plugin-handler/goto-plugins-dashboard!})
+   :ui/goto-plugins                 {:binding "t p"
+                                     :inactive (not plugin-handler/lsp-enabled?)
+                                     :fn      plugin-handler/goto-plugins-dashboard!}
 
 
-   :editor/toggle-open-blocks       {:desc    "Toggle open blocks (collapse or expand all blocks)"
-                                     :binding "t o"
+   :editor/toggle-open-blocks       {:binding "t o"
                                      :fn      editor-handler/toggle-open!}
 
-   :ui/toggle-cards                 {:desc    "Toggle cards"
-                                     :binding "t c"
+   :ui/toggle-cards                 {:binding "t c"
                                      :fn      ui-handler/toggle-cards!}
-   ;; :ui/toggle-between-page-and-file route-handler/toggle-between-page-and-file!
 
-   :git/commit                      {:desc    "Git commit message"
-                                     :binding "c"
+   :git/commit                      {:binding "c"
                                      :fn      commit/show-commit-modal!}})
 
-(defn build-category-map [symbols]
-  (reduce into {}
-          (map (fn [sym] {sym (get all-default-keyboard-shortcuts sym)}) symbols)))
+(let [keyboard-shortcuts
```

Comment:
I added these asserts so when no entries are added, the engineer is immediately aware that another data structure needs to be kept up to date